json/build.sh

```diff
@@ -90,3 +90,14 @@ go build -o json_iter_go test_jsoniter.go
 if [ ! -f /tmp/1.json ]; then
   ruby generate_json.rb
 fi
+
+if [ ! -d ./daw_json_link ]; then
```

Comment:
I'd like to centralize the vendor libraries usage here (as they seem to semi-connected). Git submodules could be an overkill, but using one directory could fit better. Something like that:
```
if [ ! -d ./daw_json_link_all ]; then
  git clone --depth 1 https://github.com/beached/daw_json_link.git ./daw_json_link_all/daw_json_link
  git clone --depth 1 https://github.com/beached/header_libraries.git ./daw_json_link_all/header_libraries
  git clone --depth 1 https://github.com/beached/utf_range.git ./daw_json_link_all/utf_range
fi
```

Reply:
Did this change and added to .gitignore